src/ShapeCrawler/Logger/SCLogger.cs

```diff
@@ -20,6 +18,7 @@ public static void Send()
     {
         return;
 
+#if false
```

Comment:
Could you please remove the `SCLogger` class and related code. In fact, it is not used.

Reply:
OK, sounds good will do. FYI, busy at work all week, so this will happen on weekend.